docs/resources/database.md

```diff
@@ -0,0 +1,49 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "bytebase_database Resource - terraform-provider-bytebase"
+subcategory: ""
+description: |-
+  The database resource.
+---
+
+# bytebase_database (Resource)
+
+The database resource. You can read or update a single database through `bytebase_database` resource.
+
+> For now we don't support create or delete the database, only support change it's project or labels. The databases're auto-sync once the instance is created.
+
+## Example Usage
+
+```terraform
+# Update a database
+resource "bytebase_database" "employee" {
```

Comment:
Do we have a way to represent the project -> database in the following way?

```
project {
...
  "instances/ins1/databases/db1",
  "instances/ins1/databases/db2",
  "instances/ins2/databases/db1",
  "instances/ins2/databases/db2",
...
}
```

Reply:
Sure, we can optimize the project data source and add the `databases` field

Reply:
Thanks! That will be easier to manage databases under a project.